modules/custom/az_paragraphs/az_paragraphs_text_media/config/install/field.field.paragraph.az_text_media.field_az_media.yml

```diff
@@ -21,9 +21,11 @@ settings:
   handler_settings:
     target_bundles:
       az_image: az_image
+      az_remote_video: az_remote_video
     sort:
       field: _none
       direction: ASC
-    auto_create: false
-    auto_create_bundle: az_image
+    auto_create: true
+    auto_create_bundle: az_remote_video
```

Comment:
Not sure if this is correct.
Should auto create any media type added, not just video